lesson_02/homework_01.py

```diff
@@ -0,0 +1,82 @@
+# task 01 == Виправте синтаксичні помилки
+print("Hello", end = ' ')
+    print("world!")

```

Comment:
Here present unexpected indent. Code starts to work only after this indent correction
Plus do not forget about linter warnings